### PR TITLE
Zoom to location on mapReady

### LINKED ISSUE #51: Handle Location in TweetFeedActivity OnStart()

### PR DESCRIPTION
**Pull request by ermel272**
### Status

`Pending`
### Description

Added common code for finding the last known current location of the device.

Handled the passing of the location information into the TweetFeedActivity.
### Changelog
- Added file LocationFinder.java for aid determining last known location
- Refactoring of code in SearchActivity.java to use LocationFinder code
- Handling of location finding logic added to TweetFeedActivity
### Addresses Issues

Closes #51

### CODE REVIEW

**Comment by srowhani** [comment]
Let's maintain a status to each PR. Is this good to go for review?

**Comment by ermel272** [comment]
@srowhani That last commit wasn't supposed to make it in... forgot that I had a PR open when I pushed to my master branch. If that last commit makes it in the map zoom will not work. Gonna get that commit out of here

**Comment by ermel272** [comment]
@srowhani Now we're good

**Comment by srowhani** [comment]
@ermel272, you haven't made any changes after that comment...

**Comment by ermel272** [comment]
@srowhani There was a commit I removed pertaining to the navigation drawer. You may not have seen it

File: app/src/main/java/com/dankideacentral/dic/SearchActivity.java [hunk 1]
```diff
@@ -209,40 +140,38 @@ private void locationButtonOnClickEvent() {
...
-        try {
-            location = LocationServices.FusedLocationApi.getLastLocation(googleApiClient);
-        } catch (SecurityException e) {
-            // SecurityException thrown if location permissions are disabled
-            // Log issue and warn user via a dialog box
-            Log.d(LOG_TAG, "Location permissions deactivated when attempting to recover location.");
-        }
+    private void getCurrentLocation() {
+        // Acquire location from the device by initializing a locationFinder object
+        locationFinder = new LocationFinder(this) {
```
Comment (srowhani) [inline app/src/main/java/com/dankideacentral/dic/SearchActivity.java:162]: This bit is pretty clever 💯

File: app/src/main/java/com/dankideacentral/dic/TweetFeedActivity.java [hunk 2]
```diff
@@ -137,10 +119,59 @@ public void mapReady(GoogleMap map, LocationManager lm, final ClusterManager cm)
...
+    }
+
+    /**
+     * Starts the twitter stream service to receive
+     * tweets at the specified latitude & longitude locations.
+     *
+     * @param latLng
+     *          The {@link LatLng} location to open the service at.
+     */
+    private void startTwitterStreamService(LatLng latLng) {
```
Comment (srowhani) [inline app/src/main/java/com/dankideacentral/dic/TweetFeedActivity.java:163]: :+1:

**Review by srowhani — APPROVED** [review]
Looks good 😀

**Pull request merged**